### DISCUSSION TITLE
update von CCU 3.71.12 auf aktuelle RaspberryMatic SW hängt

### DISCUSSION THREAD

**Discussion by thoxxxx** [OpenCCU/OpenCCU#2451] [🇩🇪 Fragen&Antworten]
Hallo,

ich habe gerade nach der Anleitung das Update von original CCU3 SW auf die aktuelle RaspberryMatic gemacht.
Leider scheint die CCU3 zu hängen. Die rote LED leuchtet dauernd.

Was ist das Problem? Was kann ich noch machen, um wieder ein laufendes System zu haben?

Übrigens, die CCU3 hat im Hintergrund Firmware auf einige Devices gesendet.

Danke und Grüße
Thomas

**Comment by thoxxxx** [comment]
Ok, hat einfach nur sehr lange gedauert.

jetzt läuft die CCU3 wieder.

Danke

**Reply by Baxxy13** [reply]
Ja, die CCU3 ist bei sowas teilweise recht lahm und stellt sich mit roter LED tot.
Das dürfte sich jetzt mit RaspberryMatic bessern.

Besser wäre es gewesen du hättest den Umzug über eine neue SD-Karte gemacht, aber gut.

Und wehe du hast Punkt 9 der Anleitung nicht befolgt. 😉

**Reply by thoxxxx** [reply]
In der Tat habe ich Punkt 9 noch nicht gemacht. Wollte erstmal schauen, wie es funktioniert.

**Reply by thoxxxx** [reply]
Wie wichtig ist denn Punkt 9?

Danke für die schnelle Antwort, übrigens!

**Reply by Baxxy13** [reply]
Nun, im hier und jetzt ist er unwichtig. Aber mit Pech stehst du dann beim nächsten RaspberryMatic Update mit Problemen da. 
Hintergrund ist das die Datenpartition für die Nutzerdaten bei CCU3 fest 2GB beträgt. Die RaspberryMatic - Updates sind aber etwas größer als die der CCU3 und so kann eventuell nicht geupdatet werden. 
Der Werksreset vergrößert die Partition auf etwa 6GB und du hast dahingehend nichts mehr zu befürchten.
Tu dir bitte den Gefallen und mache den Werksreset, am besten gleich. Schieb das nicht vor dich hin.

Nachtrag:
Du kannst ja mal auf der RM in die Systemsteuerung > Zentralenwartung gehen.
Da siehst du im oberen Bereich wieviel Platz du hast und wieviel du für ein Update brauchst.

**Reply by thoxxxx** [reply]
1,92 GB verfügbar

Ich werde den Schritt 9 heute noch machen.

Grüße
Thomas
